sdk/keyvault/azure_security_keyvault_certificates/src/generated/clients/certificate_client.rs

```diff
@@ -137,25 +138,65 @@ impl CertificateClient {
     ///   the service. The value provided should not include personally identifiable or sensitive information.
     /// * `parameters` - The parameters to create a certificate.
     /// * `options` - Optional parameters for the request.
-    pub async fn create_certificate(
+    pub fn create_certificate(
         &self,
         certificate_name: &str,
         parameters: RequestContent<CreateCertificateParameters>,
         options: Option<CertificateClientCreateCertificateOptions<'_>>,
-    ) -> Result<Response<CertificateOperation>> {
-        let options = options.unwrap_or_default();
-        let ctx = Context::with_context(&options.method_options.context);
+    ) -> Result<Poller<CertificateOperation>> {
```

Comment:
@jhendrixMSFT what do you think of the code here in terms of emitting? The line breaks I added for review readability but obviously don't have to be retained.

Reply:
Along with @antkmsft, we'll discuss this offline. I strongly believe doing more at compile time is worth it but also agree we should provide more helpers in `azure_core`. We have some already, though I don't know how well they fit this new `Poller` model compared to track 1 when they were added. But, in concept, those are good examples of what `azure_core` would provide.